selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/WorkspacesListTest.java

```diff
@@ -0,0 +1,380 @@
+  @Inject private TestProjectServiceClient testProjectServiceClient;
+  @Inject private TestWorkspaceServiceClient testWorkspaceServiceClient;
+  @Inject private SeleniumWebDriverHelper seleniumWebDriverHelper;
+  @Inject private SeleniumWebDriver seleniumWebDriver;
+  @Inject private DocumentationPage documentationPage;
+  @Inject private WorkspaceOverview workspaceOverview;
+
+  @InjectTestWorkspace(template = DEFAULT, memoryGb = 2, startAfterCreation = false)
+  private TestWorkspace blankWorkspace;
+
+  @InjectTestWorkspace(template = UBUNTU_JDK8, memoryGb = 3)
+  private TestWorkspace javaWorkspace;
+
+  private Workspaces.WorkspaceListItem expectedBlankItem;
+  private Workspaces.WorkspaceListItem expectedJavaItem;
+  private Workspaces.WorkspaceListItem expectedNewestWorkspaceItem;
+
+  @BeforeClass
+  public void setUp() throws Exception {
+    URL resource = getClass().getResource("/projects/defaultSpringProjectWithDifferentTypeOfFiles");
+
+    blankWorkspace.await();
+    javaWorkspace.await();
+
+    testProjectServiceClient.importProject(
+        javaWorkspace.getId(), Paths.get(resource.toURI()), "web-java-spring", MAVEN_SPRING);
+
+    expectedBlankItem =
+        new Workspaces.WorkspaceListItem(
+            defaultTestUser.getName(),
+            blankWorkspace.getName(),
+            BLANK_WS_MB,
+            BLANK_WS_PROJECTS_COUNT);
+    expectedJavaItem =
+        new Workspaces.WorkspaceListItem(
+            defaultTestUser.getName(), javaWorkspace.getName(), JAVA_WS_MB, JAVA_WS_PROJECTS_COUNT);
+
+    expectedNewestWorkspaceItem =
+        new Workspaces.WorkspaceListItem(
+            defaultTestUser.getName(),
+            NEWEST_CREATED_WORKSPACE_NAME,
+            BLANK_WS_MB,
+            BLANK_WS_PROJECTS_COUNT);
+
+    dashboard.open();
+  }
+
+  @BeforeMethod
+  public void prepareToTestMethod() {
+    dashboard.waitDashboardToolbarTitle();
+    dashboard.selectWorkspacesItemOnDashboard();
+  }
+
+  @AfterClass
+  public void tearDown() throws Exception {
+    deleteIfWorkspaceExist(expectedNewestWorkspaceItem.getWorkspaceName());
+  }
+
+  @Test
+  public void shouldDisplayElements() {
+    workspaces.waitPageLoading();
+    dashboard.waitWorkspacesCountInWorkspacesItem(EXPECTED_WORKSPACES_COUNT);
+
+    List<Workspaces.WorkspaceListItem> items = workspaces.getVisibleWorkspaces();
+    assertEquals(
+        workspaces.getWorkspacesListItemByWorkspaceName(
+            items, expectedBlankItem.getWorkspaceName()),
+        expectedBlankItem);
+
+    assertEquals(
+        workspaces.getWorkspacesListItemByWorkspaceName(items, expectedJavaItem.getWorkspaceName()),
+        expectedJavaItem);
+  }
+
+  @Test
+  public void checkWorkspaceSelectingByCheckbox() throws Exception {
+    String blankWorkspaceName = blankWorkspace.getName();
+    String javaWorkspaceName = javaWorkspace.getName();
+
+    workspaces.waitPageLoading();
+    workspaces.selectAllWorkspacesByBulk();
+
```

Comment:
it's better not to divide `selectAllWorkspacesByBulk()` and `assertTrue` statements by empty row to make separate steps more obvious.